bin/init.bash

```diff
@@ -295,8 +300,16 @@ set_nginx_config() {
     replace_set_me "${file}" '.ingressNginx.controller.service.enabled' "${service_enabled}"
     replace_set_me "${file}" '.networkPolicies.global.externalLoadBalancer' "${external_load_balancer}"
     replace_set_me "${file}" '.networkPolicies.global.ingressUsingHostNetwork' "${ingress_using_host_network}"
-    replace_set_me "${file}" '.ingressNginx.controller.service.allocateLoadBalancerNodePorts' 'true'
+    replace_set_me "${file}" '.ingressNginx.controller.service.allocateLoadBalancerNodePorts' "${allocate_loadbalancer_node_ports}"
 
+    if [ "${CK8S_CLOUD_PROVIDER}" = "azure" ]; then
+        replace_set_me "${file}" '.externalTrafficPolicy.local' 'true'
+        replace_set_me "${file}" '.networkPolicies.ingressNginx.ingressOverride.enabled' 'true'
+        replace_set_me "${file}" '.networkPolicies.ingressNginx.ingressOverride.ips' "[ 0.0.0.0/0 ]"
```

Comment:
The network policy bug is that this is currently being replaced in common but it should be replaced in the sc and wc configs

Reply:
Yes, but it's not a `set-me` value in that config file.

Reply:
Weirdly enough, when i add `ips: set-me` i get bad expression error, but without it, the init works fine.

Reply:
Sent a fix for this here: https://github.com/elastisys/compliantkubernetes-apps/pull/2095